ws.py

```diff
@@ -80,12 +79,21 @@
 
 if "ChatExchangeU" in os.environ:
     username = os.environ["ChatExchangeU"]
+elif GlobalVars.chatexchange_u:
+    log('info', "ChatExchange username loaded from config")
+    username = GlobalVars.chatexchange_u
 else:
-    username = input("Username: ")
+    log('error', "No ChatExchange username provided. Set it in config or provide it via environment variable")
+    os._exit(6)
+
 if "ChatExchangeP" in os.environ:
     password = os.environ["ChatExchangeP"]
```

Comment:
Can a similar `info` log be added here so that we know they were loaded from environment variables?